helm/nats-operator/templates/deployment.yaml

```diff
@@ -26,12 +32,6 @@ spec:
         {{ .Values.podAnnotations }}
       {{- end }}
     spec:
-      updateStrategy:
-        type: {{ .Values.updateStrategy }}
-        {{- if .Values.rollingUpdatePartition }}
-        rollingUpdate:
-          partition: {{ .Values.rollingUpdatePartition }}
```

Comment:
I think this is a StatefulSet concept and can be removed from the definition. Can't find references for it outside of the StatefulSet documentation.

https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#partitions